### PR TITLE
Adding hidecbar option to scatter

### PR DESCRIPTION
**Pull request by mtewes**

### CODE REVIEW

File: megalut/plot/scatter.py [hunk 1]
```diff
@@ -128,10 +130,12 @@ def scatter(ax, cat, featx, featy, featc=None, cmap="jet", title=None, text=None
 			ax.errorbar(data[featx.colname], data[featy.colname], xerr=xerr, yerr=yerr, **myerrorbarkwargs)
 		
 		stuff = ax.scatter(data[featx.colname], data[featy.colname], c=data[featc.colname], **mykwargs)
-		divider = make_axes_locatable(ax)
-		cax = divider.append_axes("right", "5%", pad="3%")
-		cax = plt.colorbar(stuff, cax)
-		cax.set_label(featc.nicename)
+		
+		if hidecbar is False:
```
Comment (kuntzer) [inline megalut/plot/scatter.py]: Purely geeky question: is that better than 

    if not hidecbar:

?

Comment (mtewes) [inline megalut/plot/scatter.py]: There might be pros and cons, but I didn't really think about these here :)

Comment (kuntzer) [inline megalut/plot/scatter.py]: I guessed so :) Fine by me btw

Comment (mtewes) [inline megalut/plot/scatter.py]: Okay, thanks for checking. BTW, don't you have similar stuff in the fiducial branch that we could merge into master ?

Comment (kuntzer) [inline megalut/plot/scatter.py]: Not sure, but I could merge fiducial once more, I'll make sure it's up to date before PL-ing

**Pull request merged**